Gems/Atom/RHI/WebGPU/Code/Source/RHI.Builders/ShaderPlatformInterface.cpp

```diff
@@ -90,20 +96,237 @@ namespace AZ
             {
                 return MacAzslShaderHeader;
             }
+            else if (platform.m_identifier == "linux")
+            {
+                return LinuxAzslShaderHeader;
+            }
             else
             {
                 return WindowsAzslShaderHeader;
             }
         }
 
         bool ShaderPlatformInterface::CompilePlatformInternal(
```

Comment:
I wonder if there is a way to refactor this code to somewhere common for multiple RHIs. Not needed at this point but more so for future. I imagine this code will change further to incorporate WGSL.